reference/data_solutions.yaml

```diff
@@ -0,0 +1,411 @@
+          description: The requested resource was not found. Please specify valid Web Analytic ID.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse404'
+    put:
+      tags:
+        - Analytics
+      operationId: updateAnalytic
+      summary: Updates a Web Analytic
+      description: 'Updates a single Web Analytics data for a specified channel. If `channel_id` is not set, analytic data updates for a default channel.'
+      parameters:
+        - name: channel_id
+          in: query
+          description: 'Channel ID. If empty, the default Channel ID will be used.'
+          schema:
+            type: integer
+        - name: id
+          in: path
+          description: Web Analytic ID.
+          required: true
+          schema:
+            type: integer
+        - name: store_hash
+          in: path
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: A JSON Web Analytic object
+        content:
+          application/json:
+            schema:
+              oneOf:
+                - $ref: '#/components/schemas/AnalyticGoogle'
+                - $ref: '#/components/schemas/AnalyticVisualWebsiteOptimizer'
+                - $ref: '#/components/schemas/AnalyticFacebookPixel'
+                - $ref: '#/components/schemas/AnalyticSegment'
+                - $ref: '#/components/schemas/AnalyticSiteVerificationTags'
+                - $ref: '#/components/schemas/AnalyticAffiliateConversionTracking'
+      responses:
+        '200':
+          description: Successful operation.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Analytic'
+        '400':
+          description: Bad request. Authentication Required.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse400'
+        '404':
+          description: The requested resource was not found. Please specify valid Web Analytic ID.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse404'
+        '409':
+          description: Web Analytic was in conflict with another Web Analytic. This can be caused by duplicated unique values (such as ID).
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse409'
+        '422':
+          description: 'Web Analytic data was not valid. This is the result of missing required fields, or of invalid data. See the response for more details.'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse422'
+components:
+  schemas:
+    Analytics:
+      type: object
+      properties:
+        data:
+          type: array
+          items:
```

Comment:
There is an issue with the property type of `items`, but how we solve it will depend on whether each analytic shares the same property keys or not.

Reply:
Each analytic has different property keys. How can I fix the error?

Reply:
Have fixed the change based on your feedback in slack